Rise-Vision/widget-google-calendar
# Prevent day event from being treated as multi day event

## Description
Check if event is current day and if the end date is next day but within 24 hours, handle the event as a day event and not a multi-day event

## Motivation and Context
This fixes issue #135 . Any calendar that has an event that starts today but ends tomorrow should display it as a day event and include the time. When it's a multi-day event, time is omitted, hence time has not been showing for this scenario. 

## How Has This Been Tested?
This has been tested locally and with a staged widget in a presentation. Validated in Preview and on ChrOS display. 

https://apps.risevision.com/editor/workspace/c26a564f-5adf-431f-92ea-5318bb00ce4f?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
  - Deploy Tuesday, Aug 6th as I am on vacation until then
  - Manually tested
  - Added automated integration tests
  - Once deployed, will validate on production with a test presentation & display. Will also take 5 random user displays (obtained from BQ) that have Calendar content and restart their displays to validate no issues.
  - Monitor calendar logs for remainder of day after deployment
  - Rollback is done by re-running last CCI master merge (production deployment)
  - Support will be made aware of fix deployment
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
- No documentation required

cc: @alanclayton
